pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -52,6 +52,18 @@
 DocumentFormat: TypeAlias = Literal['csv', 'doc', 'docx', 'html', 'md', 'pdf', 'txt', 'xls', 'xlsx']
 VideoFormat: TypeAlias = Literal['mkv', 'mov', 'mp4', 'webm', 'flv', 'mpeg', 'mpg', 'wmv', 'three_gp']
 
+# OpenTelemetry GenAI finish reasons used for gen_ai.response.finish_reasons
+# See mappings in provider implementations (e.g., OpenAI/Google) for how vendor reasons map here.
+OtelFinishReason: TypeAlias = Literal[
+    'stop',
+    'length',
+    'content_filter',
+    'timeout',
+    'cancelled',
+    'error',
+    'other',
```

Comment:
See https://github.com/pydantic/pydantic-ai/pull/2590#discussion_r2325590112 for the new values, can you update this and the mappers please?